### PR TITLE
♻️  Clarify possible combinations of PostgreSQL service types and plans

### PR DESCRIPTION
**Pull request by TimMoser92**
This morning I discovered that I missunderstood the available postgresql plans. I read the documentation as saying "postgresql11 has only the shared plan and postgresql13 has only the RDS standard plans". This was mostly caused by me misreading the sentence 
> "With the `shared` plan (only supported __for type__ `postgresql11`)"... (original, correct)
as
> "With the `shared` plan (only supported __type for__ `postgresql11`)"... (my mistaken reading)

But the misunderstanding was reinforced further by the two other passages, which I propose to change here.

### CODE REVIEW

File: content/latest/user/configuration/services.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ We currently support the Service types described in the table below. Refer to th
 
 | Type | Plan(s) | Options | Link |
 |------|---------|---------|-------|
-| [`postgresql11`]({{< relref "#postgresql" >}}) | `shared` | [`extensions`]({{< relref "#postgresql-extensions" >}}) | Environment Variable |
+| [`postgresql11`]({{< relref "#postgresql" >}}) | `shared` or [RDS PostgreSQL node type](https://aws.amazon.com/de/rds/postgresql/pricing/) | [`extensions`]({{< relref "#postgresql-extensions" >}}) | Environment Variable |
```
Comment (kevinscholz) [inline content/latest/user/configuration/services.md:37]: This should actually not be possible right now, unfortunately. We have a ticket in our backlog to support both plan types for both versions.

File: content/latest/user/configuration/services.md [hunk 2]
```diff
@@ -192,7 +192,7 @@ Your database user will own the database and all objects within the database.
...
 
-If you require a dedicated database instance for complience, performance or reliability, the plan value can be set to any valid [RDS PostgreSQL node type](https://aws.amazon.com/elasticsearch-service/pricing/) when using type `postgresql13`. `db.t3.micro` is a sensible default for small workloads. To enable the _Multi-AZ_ feature, which keeps a hot standby instance in a second availability zones for higher reliability, add `.ha` at the end of the node type: e.g. `db.t3.micro.ha` for the _Multi-AZ_ version of `db.t3.micro`. Using plans with `.ha` doubles the cost of the database instance.
+If you require a dedicated database instance for complience, performance or reliability, the plan value can be set to any valid [RDS PostgreSQL node type](https://aws.amazon.com/elasticsearch-service/pricing/). `db.t3.micro` is a sensible default for small workloads. To enable the _Multi-AZ_ feature, which keeps a hot standby instance in a second availability zones for higher reliability, add `.ha` at the end of the node type: e.g. `db.t3.micro.ha` for the _Multi-AZ_ version of `db.t3.micro`. Using plans with `.ha` doubles the cost of the database instance.
```
Comment (kevinscholz) [inline content/latest/user/configuration/services.md:195]:
```suggestion
If you require a dedicated database instance for compliance, performance or reliability, the type must currently be set to `postgresql13` and the plan value can be set to any valid [RDS PostgreSQL node type](https://aws.amazon.com/elasticsearch-service/pricing/). `db.t3.micro` is a sensible default for small workloads. To enable the _Multi-AZ_ feature, which keeps a hot standby instance in a second availability zones for higher reliability, add `.ha` at the end of the node type: e.g. `db.t3.micro.ha` for the _Multi-AZ_ version of `db.t3.micro`. Using plans with `.ha` doubles the cost of the database instance.
```

**Review by kevinscholz — CHANGES_REQUESTED** [review]
Thanks for sharing that this part of the documentation is not perfectly explaining which options can be used together. 👍 

I think there is still a misunderstanding though – please see my comments.

**Comment by TimMoser92** [comment]
Thanks for the reply and clarification. Then my initial understanding was of the documentation was correct and the misunderstanding occurred only after reading it twice. I think your proposal will prevent similar confusion in the future. 👍

**Review by kevinscholz — APPROVED** [review]

**Pull request merged**